.github/workflows/ci.yml

```diff
@@ -11,46 +11,56 @@ jobs:
       matrix:
         include:
           - os: ubuntu-22.04
-            rev: nightly/nvim-linux64.tar.gz
+            rev: nightly
             manager: sudo apt-get
             packages: -y ripgrep
           - os: ubuntu-22.04
-            rev: v0.9.0/nvim-linux64.tar.gz
+            rev: v0.9.0
             manager: sudo apt-get
             packages: -y ripgrep
           - os: macos-12
-            rev: nightly/nvim-macos.tar.gz
+            rev: nightly
             manager: brew
             packages: ripgrep
           - os: macos-12
-            rev: v0.9.0/nvim-macos.tar.gz
+            rev: v0.9.0
             manager: brew
             packages: ripgrep
+          - os: windows-2022
+            rev: nightly
+            manager: choco
+            packages: ripgrep
+          - os: windows-2022
+            rev: v0.9.0
+            manager: choco
+            packages: ripgrep
+
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
+
       - run: date +%F > todays-date
+
+      - uses: rhysd/action-setup-vim@v1
+        with:
+          neovim: true
+          version: ${{ matrix.rev }}
```

Comment:
i am not the biggest fan of using third party actions for this simple stuff, imo it "doesnt give us" that much and we are dependent on that action to be maintained when issues arise. your linked issue is the perfect example. we cant fix something like this ourselves. but yeah we can still do that

Reply:
Yeah I know what you mean.
For what it's worth, there are some other notable vim/neovim plugins using this action and this issue of the neovim nightly issue on macos got resolved in like 3 days. Our macos nightly job has been broken just as long and still is.

Reply:
yeah sure. just saying not happy with stuff like that, because we cant act. Not that we shouldnt do it at all. windows support probably justifies it.